scripts/stackstorm.js

```diff
@@ -162,6 +169,37 @@ module.exports = function(robot) {
     );
   };
 
+  var getExecutionHistoryUrl = function(execution_id) {
```

Comment:
I know this is an anti-pattern since it makes things hard to test and less re-usable, but I just followed the existing style.

After this PR is merged, I will open a separate one where I will refactor stackstorm.js module. I will move most of the functions to the top-level (outside of the closure), export them and add tests. This will also make code more re-usable.